packages/generator/src/artifacts/css/parser-css.ts

```diff
@@ -29,7 +29,7 @@ export const generateParserCss = (ctx: Context) => (result: ParserResultType) =>
 
       result.jsx.forEach((jsx) => {
         jsx.data.forEach((data) => {
-          sheet.processStyleProps(data)
+          sheet.processStyleProps(filterProps(ctx, data))
```

Comment:
It seems like this filterProps should be happening in the processStyleProps method of the stylesheet.

Can we move it there to avoid the duplicates?